packages/components/src/DataList/components/DataListHeaderTile/DataListHeaderTile.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import classnames from "classnames";
+import styles from "./DataListHeaderTile.css";
+import { DataListHeader, DataListObject } from "../../DataList.types";
+import { Text } from "../../../Text";
+import { Icon } from "../../../Icon";
+import { useDataListContext } from "../../context/DataListContext";
+
+interface DataListHeaderTileProps<T extends DataListObject> {
+  headers: DataListHeader<T>;
+  headerKey: string;
+}
+
+export function DataListHeaderTile<T extends DataListObject>({
+  headers,
+  headerKey,
+}: DataListHeaderTileProps<T>) {
+  const { sorting } = useDataListContext();
+
+  const isSortable = sorting?.sortable.includes(headerKey);
+  const sortingState = sorting?.state;
+
+  function toggleSorting(sortingKey: string) {
+    if (sortingKey === sortingState?.key) {
+      sorting?.onSort({
+        direction: sortingState.direction === "asc" ? "desc" : "asc",
+        key: sortingKey,
+      });
+    } else {
+      sorting?.onSort({ direction: "asc", key: sortingKey });
+    }
+  }
+
+  return (
+    <div
+      className={classnames(styles.headerLabel, {
+        [styles.sortable]: isSortable,
+      })}
+      onClick={isSortable ? () => toggleSorting(headerKey) : undefined}
+    >
+      <Text variation="subdued" maxLines="single" size="small">
+        {headers[headerKey]}
+      </Text>
+      {sortingState?.key === headerKey && (
+        <SortingArrows order={sortingState.direction} />
+      )}
+    </div>
+  );
+}
+
+function SortingArrows({ order }: { order: "asc" | "desc" | "none" }) {
+  if (order === "none") return <></>;
+
+  return (
+    <div>
+      <span className={styles.arrowUp}>
+        <Icon
+          name="arrowUp"
+          color={order === "asc" ? "greyBlue" : "blue"}
```

Comment:
since this and line 66 pretty much have the same logic except for the value, can we simplify this and avoid that ternary?

Reply:
I don't know... maybe I could extract the colors to constants called `activeColor` and `inactivateColor`, but the ternary is very straightforward and works well for inverting the colors depending on the order. Do you have any suggestions?